src/app/pages/main/pages/home/home.component.html

```diff
@@ -1,3 +1,8 @@
-<app-request-wrapper #postsData [request$]="postsData$">
-  <app-post *ngFor="let item of postsData.data" class="mb-5" [post]="getPost | executeWith : item"></app-post>
-</app-request-wrapper>
+<!--<app-request-wrapper #postsData [request$]="postsData$">-->
+<!--  <app-post *ngFor="let item of postsData.data" class="mb-5" [post]="getPost | executeWith : item"></app-post>-->
+<!--</app-request-wrapper>-->
```

Comment:
Major: Commented out code snippets should be removed if they are not needed.

Reply:
Done